cli-output/src/lib.rs

```diff
@@ -1,3 +1,15 @@
 mod cli_output;
 pub mod display;
 pub use cli_output::*;
+
+pub trait QuietDisplay: std::fmt::Display {
+    fn write_str(&self, w: &mut dyn std::fmt::Write) -> std::fmt::Result {
```

Comment:
Should we use `std::io::Write` here instead?  If we did, we could normalize all `OutputFormat` types over it and avoid going to `String` when we don't need to

Reply:
Yes, I think I'd prefer that, if it's not too painful. Good idea!

Reply:
This wound up a little uglier than I was initially thinking.  For whatever reason I thought `String` implemented `std::io::Write`, which makes no sense given the UTF-8 constraint.

If we decide to move forward, I'll squash this in and generalize over `OutputFormat`

Reply:
Oh duh. I was thinking that too.
I'm pretty ambivalent now that I see it, compared to the simplicity of your original implementation. Your call.

Reply:
I backed it out. Not enough improvement to justify the ugly, IMO